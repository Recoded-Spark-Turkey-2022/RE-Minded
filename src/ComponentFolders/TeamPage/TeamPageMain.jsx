import React from 'react';
import { useTranslation } from 'react-i18next';
import TeamCards from './TeamCards';
import menar from './Images/Menar Selamet_Turkey_2022.jpg';
import sohail from './Images/SohailAhmed.jpeg';
import seyma from './Images/Seyma Demir_Turkey_2022.jpg';
import selvi from './Images/Selvi Ece Dugan_Turkey_2022.jpg';
import ammar from './Images/Ammar Eldik_Turkey_2022.jpg';

// name and jobs for pass data to card with props
// do we need to show team trainer?
const names = [
  'Menar Selamet',
  'Sohail Ahmed',
  'Åžeyma Demir',
  'Selvi Ece Dugan',
  'Ammar Eldik',
];
const jobs = ['Lead Engineer', 'Frontend Developer'];
const photos = [menar, sohail, seyma, selvi, ammar];

function TeamPageMain() {

  const { t } = useTranslation();

  return (
    <div className="flex flex-col text-[#424A4F] mb-56">
      <div className=" font-poppins lg:text-5xl text-3xl leading-10">
        <h1 className="lg:ml-44 lg:mt-20 ml-10 sm:mr-10 md:mr-10 mt-10 text-black">
          {t('team.h1')}
        </h1>
      </div>
      <div className="text-3xl leading-10">
        <h3 className="lg:ml-44 lg:mt-28 ml-10 mt-8">{t('team.meet')}</h3>
      </div>
      <div className=" ml-36 mt-[4em] mr-36">
        <div className="font-poppins flex flex-rows justify-items-center justify-evenly flex-wrap -m-6">
          {names.map((name, index) => {
            return (
              <TeamCards name={name} job={jobs[1]} photo={photos[index]} />
            );
          })}
        </div>
      </div>
    </div>
  );
}

export default TeamPageMain;
