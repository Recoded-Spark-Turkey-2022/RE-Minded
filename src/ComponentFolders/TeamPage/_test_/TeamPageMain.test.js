import React from 'react';
import renderer from 'react-test-renderer';
import TeamPageMain from '../TeamPageMain';

test('Team Page snapshot test', () => {
  const component = renderer.create(<TeamPageMain />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});