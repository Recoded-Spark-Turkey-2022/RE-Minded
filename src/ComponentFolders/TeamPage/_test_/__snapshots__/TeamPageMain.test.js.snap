// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Team Page snapshot test 1`] = `
<div
  className="flex flex-col text-[#424A4F] "
>
  <div
    className=" font-poppins lg:text-5xl text-3xl leading-10"
  >
    <h1
      className="lg:ml-44 lg:mt-20 ml-10 sm:mr-10 md:mr-10 mt-10 text-black"
    >
      WE ARE HEALING, NICE TO MEET YOU!
    </h1>
  </div>
  <div
    className="text-3xl leading-10"
  >
    <h3
      className="lg:ml-44 lg:mt-28 ml-10 mt-8"
    >
      Meet the Team!
    </h3>
  </div>
  <div
    className=" ml-36 mt-[4em] mr-36"
  >
    <div
      className="font-poppins flex flex-rows justify-items-center justify-evenly flex-wrap -m-6"
    >
      <div
        className="bg-[#EAF8F9] rounded-[2em] shadow-md lg:max-w-[17em] mt-[4%] lg:mt-[0%]"
      >
        <img
          alt="photoOfTeamMember"
          className="object-cover w-full rounded-t-[2em] border-b-[0.5em] border-[#6BD24D] h-64"
          src="Menar Selamet_Turkey_2022.jpg"
        />
        <div
          className="p-4 flex flex-col items-center"
        >
          <h4
            className="text-2xl font-semibold leading-5 text-[#424A4F] "
          >
            Menar Selamet
          </h4>
          <p
            className="text-lg mb-2 leading-10"
          >
            Frontend Developer
          </p>
        </div>
      </div>
      <div
        className="bg-[#EAF8F9] rounded-[2em] shadow-md lg:max-w-[17em] mt-[4%] lg:mt-[0%]"
      >
        <img
          alt="photoOfTeamMember"
          className="object-cover w-full rounded-t-[2em] border-b-[0.5em] border-[#6BD24D] h-64"
          src="SohailAhmed.jpeg"
        />
        <div
          className="p-4 flex flex-col items-center"
        >
          <h4
            className="text-2xl font-semibold leading-5 text-[#424A4F] "
          >
            Sohail Ahmed
          </h4>
          <p
            className="text-lg mb-2 leading-10"
          >
            Frontend Developer
          </p>
        </div>
      </div>
      <div
        className="bg-[#EAF8F9] rounded-[2em] shadow-md lg:max-w-[17em] mt-[4%] lg:mt-[0%]"
      >
        <img
          alt="photoOfTeamMember"
          className="object-cover w-full rounded-t-[2em] border-b-[0.5em] border-[#6BD24D] h-64"
          src="Seyma Demir_Turkey_2022.jpg"
        />
        <div
          className="p-4 flex flex-col items-center"
        >
          <h4
            className="text-2xl font-semibold leading-5 text-[#424A4F] "
          >
            Åžeyma Demir
          </h4>
          <p
            className="text-lg mb-2 leading-10"
          >
            Frontend Developer
          </p>
        </div>
      </div>
      <div
        className="bg-[#EAF8F9] rounded-[2em] shadow-md lg:max-w-[17em] mt-[4%] lg:mt-[0%]"
      >
        <img
          alt="photoOfTeamMember"
          className="object-cover w-full rounded-t-[2em] border-b-[0.5em] border-[#6BD24D] h-64"
          src="Selvi Ece Dugan_Turkey_2022.jpg"
        />
        <div
          className="p-4 flex flex-col items-center"
        >
          <h4
            className="text-2xl font-semibold leading-5 text-[#424A4F] "
          >
            Selvi Ece Dugan
          </h4>
          <p
            className="text-lg mb-2 leading-10"
          >
            Frontend Developer
          </p>
        </div>
      </div>
      <div
        className="bg-[#EAF8F9] rounded-[2em] shadow-md lg:max-w-[17em] mt-[4%] lg:mt-[0%]"
      >
        <img
          alt="photoOfTeamMember"
          className="object-cover w-full rounded-t-[2em] border-b-[0.5em] border-[#6BD24D] h-64"
          src="Ammar Eldik_Turkey_2022.jpg"
        />
        <div
          className="p-4 flex flex-col items-center"
        >
          <h4
            className="text-2xl font-semibold leading-5 text-[#424A4F] "
          >
            Ammar Eldik
          </h4>
          <p
            className="text-lg mb-2 leading-10"
          >
            Frontend Developer
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
